clang/runtime/dpct-rt/include/dpct/image.hpp

```diff
@@ -471,48 +471,89 @@ class sampling_info {
   sycl::filtering_mode _filtering_mode = sycl::filtering_mode::nearest;
   sycl::coordinate_normalization_mode _coordinate_normalization_mode =
       sycl::coordinate_normalization_mode::unnormalized;
+  sycl::filtering_mode _mipmap_filtering = sycl::filtering_mode::nearest;
+  float _min_mipmap_level_clamp = 0.f;
```

Comment:
the new added 4 attributes is not so straightforward,  please add comment on it and it's get/set interface.

Reply:
fixed